FE/src/components/Dashboard.tsx

```diff
@@ -3,30 +3,29 @@ import { useSelector } from 'react-redux';
 import { fetchMovies } from '../redux/movieSlice';
 import image from '../styles/images/dashboard-bg.jpg';
 import Grid from '@mui/system/Unstable_Grid';
-import Movies from './Movies';
+import Movies from './Movies/Movies';
 import { useAppDispatch } from '../redux/reduxHooks';
 
 const Dashboard = () => {
   const dispatch = useAppDispatch();
-  const state = useSelector((state: any) => state.movies);
-  const { movies } = state;
-
-  console.log(movies);
+  const movies = useSelector((state: any) => state.movies.movies);
 
   React.useEffect(() => {
     dispatch(fetchMovies());
   }, [dispatch]);
 
   return (
     <Grid
+      container
+      xs={12}
       sx={{
-        // overflow: 'scroll',
+        overflow: 'scroll',

```

Comment:
Overflow scroll will always show the scroll bar, meanwhile 'auto' will hide it when it is not required.